### DISCUSSION TITLE
Migration from Akka

### DISCUSSION THREAD

**Discussion by deabreu** [apache/pekko-http#255] [General]
Hello, community. Are there any migration practices to migrate from Akka to Pekko?

**Comment by pjfanning** [comment]
please read https://pekko.apache.org/docs/pekko-http/current/migration-guide/index.html

**Comment by deabreu** [comment]
Hi, the install guide has bad information.

I cannot resolve dependency for Pekko HTTP version "1.0.0-RC1+1-d9534a93-SNAPSHOT" as informed. Can you provide an accurate resolver, please?

**Reply by mdedetrich** [reply]
Snapshots are published to Apache Snapshots Repo http://repository.apache.org/snapshots/, see https://infra.apache.org/repository-faq.html#basic. If you are using SBT 1.9.0 or above you can use the following

```scala
ThisBuild / resolvers += Resolver.ApacheMavenSnapshotsRepo
```
